test/api-integration-test/run_test.sh

```diff
@@ -64,13 +64,12 @@ cd "${BASE_DIR}/${TEST_DIR}"
 export GO111MODULE=on
 
 echo "Run integration test..."
-TEST_RESULT=`go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true 2>&1`
-TEST_EXIT_CODE=$?
+LOG_FILE=$(mktemp)
+TEST_EXIT_CODE=0 # reference for how to save exit code: https://stackoverflow.com/a/18622662
+go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true |& tee $LOG_FILE || TEST_EXIT_CODE=$?
```

Comment:
Are you sure `$?` will be the exit code of `go` and not `tee`?

Reply:
Thanks for the catch! You are right. tee affects exit code.
Added `set -o pipefail`